EXTRAS/reports/aruba

```diff
@@ -1030,6 +1030,282 @@
+                        dot1qConstraintVlan(1) type=15 hint=d
+                        dot1qConstraintSet(2) type=16
+                        dot1qConstraintType(3) type=3
+                        dot1qConstraintStatus(4) type=3
+                    dot1qConstraintSetDefault(9) type=16
+                    dot1qConstraintTypeDefault(10) type=3
+                  dot1vProtocol(5) type=0
+                    dot1vProtocolGroupTable(1) type=0
+                      dot1vProtocolGroupEntry(1) type=0
+                        dot1vProtocolTemplateFrameType(1) type=3
+                        dot1vProtocolTemplateProtocolValue(2) type=2
+                        dot1vProtocolGroupId(3) type=16
+                        dot1vProtocolGroupRowStatus(4) type=3
+                    dot1vProtocolPortTable(2) type=0
+                      dot1vProtocolPortEntry(1) type=0
+                        dot1vProtocolPortGroupId(1) type=16
+                        dot1vProtocolPortGroupVid(2) type=16
+                        dot1vProtocolPortRowStatus(3) type=3
+                qBridgeConformance(2) type=0
+                  qBridgeGroups(1) type=0
+                    qBridgeBaseGroup(1) type=22
+                    qBridgeFdbUnicastGroup(2) type=22
+                    qBridgeFdbMulticastGroup(3) type=22
+                    qBridgeServiceRequirementsGroup(4) type=22
+                    qBridgeFdbStaticGroup(5) type=22
+                    qBridgeVlanGroup(6) type=22
+                    qBridgeVlanStaticGroup(7) type=22
+                    qBridgePortGroup(8) type=22
+                    qBridgeVlanStatisticsGroup(9) type=22
+                    qBridgeVlanStatisticsOverflowGroup(10) type=22
+                    qBridgeVlanHCStatisticsGroup(11) type=22
+                    qBridgeLearningConstraintsGroup(12) type=22
+                    qBridgeLearningConstraintDefaultGroup(13) type=22
+                    qBridgeClassificationDeviceGroup(14) type=22
+                    qBridgeClassificationPortGroup(15) type=22
+                    qBridgePortGroup2(16) type=22
+                  qBridgeCompliances(2) type=0
+                    qBridgeCompliance(1) type=26
+                    qBridgeCompliance2(2) type=26
+              dot1dConformance(8) type=0
+                dot1dGroups(1) type=0
+                  dot1dBaseBridgeGroup(1) type=22
+                  dot1dBasePortGroup(2) type=22
+                  dot1dStpBridgeGroup(3) type=22
+                  dot1dStpPortGroup(4) type=22
+                  dot1dStpPortGroup2(5) type=22
+                  dot1dStpPortGroup3(6) type=22
+                  dot1dTpBridgeGroup(7) type=22
+                  dot1dTpFdbGroup(8) type=22
+                  dot1dTpGroup(9) type=22
+                  dot1dStaticGroup(10) type=22
+                  dot1dNotificationGroup(11) type=23
+                dot1dCompliances(2) type=0
+                  bridgeCompliance1493(1) type=26
+                  bridgeCompliance4188(2) type=26
```

Comment:
Strange, RFC related objects in the "reports/aruba" diff that are not in the "reports/all" ? Haven't checked in detail yet, perhaps these are copies of RFC MIBs that are relocated somewhere into the aruba tree for experimental/testing reasons?

Reply:
good catch, i guess these might be due to new mibs importing rfc mibs?

ARUBAWIRED-CIPT-MIB:

        VlanIndex
            FROM Q-BRIDGE-MIB